### DISCUSSION TITLE
Add a non-minified version of dist/js/winbox.min.js

### DISCUSSION THREAD

**Discussion by kodie** [nextapps-de/winbox#140] [Ideas]
When adding winbox to a project that can't use ES6 modules, the only option is to use `dist/js/winbox.min.js`, but if we're concatting that file into a file that ultimately gets minified, winbox gets minified again which isn't good. It would be really nice to have a non-minified version of this file available (all modules bundled in but not minified), so `dist/js/winbox.js`.

**Comment by ts-thomas** [comment]
> but if we're concatting that file into a file that ultimately gets minified, winbox gets minified again which isn't good

Could you explain me why this isn't good? Did you use something like Closure Compiler Advanced which renames properties?
